### PR TITLE
feat: generate schema from openapi spec

### PR DESCRIPTION
**Pull request by Swolebrain**
This feature allows users to generate a schema from an openApi spec in the same way that the Verified Permissions console does it.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -59,6 +59,30 @@ const policyStore = new PolicyStore(scope, "PolicyStore", {
 });
 ```
 
+## Schemas
+
+If you want to have type safety when defining a schema, you can accomplish this in typescript. Simply use the `Schema` type exported by the `@cedar-policy/cedar-wasm`.
+
+You can also generate a simple schema from a swagger file using the static function `schemaFromOpenApiSpec` in the PolicyStore consrtuct. This functionality replicates what you can find in the AWS Verified Permissions console.
```
Comment (svenjaraether) [inline README.md:66]:
```suggestion
You can also generate a simple schema from a swagger file using the static function `schemaFromOpenApiSpec` in the PolicyStore construct. This functionality replicates what you can find in the AWS Verified Permissions console.
```

File: src/policy-store.ts [hunk 2]
```diff
@@ -254,6 +254,45 @@ export class PolicyStore extends PolicyStoreBase {
     return new Import(policyStoreArn, policyStoreId);
   }
 
+  /**
+   * This method generates a schema based on an swagger file. It makes the same assumptions and decisions
```
Comment (svenjaraether) [inline src/policy-store.ts:258]:
```suggestion
   * This method generates a schema based on an swagger file in JSON format. It makes the same assumptions and decisions
```

File: src/policy-store.ts [hunk 3]
```diff
@@ -254,6 +254,45 @@ export class PolicyStore extends PolicyStoreBase {
...
+   * made in the Amazon Verified Permissions console. This feature is built for swagger files generated from an ApiGateway
```
Comment (svenjaraether) [inline src/policy-store.ts:259]:
```suggestion
   * made in the Amazon Verified Permissions console. This feature is built for swagger files generated from an Amazon API Gateway
```

**Review by reste85 — APPROVED** [review]
LGTM, thank you!

**Pull request merged**